dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -139,7 +132,7 @@ export const enhanceCollections = ({
 			},
 			canShowMore: hasMore && !collection.config.hideShowMore,
 			targetedTerritory: collection.targetedTerritory,
-			collectionBranding: sponsorBranding,
+			collectionBranding: decideCollectionBranding(allCards, editionId),
```

Comment:
The aim is to push all the logic for collection branding in this method and add unit tests. For now it only returns `Branding` of `Sponsored` type.